3ds2playground/src/main/kotlin/com/stripe/android/stripe3ds2playground/MainActivity.kt

```diff
@@ -471,7 +472,9 @@ class MainActivity : AppCompatActivity() {
                     acsTransId = acsTransId,
                     threeDsServerTransId = threeDsServerTransId,
                     sdkTransId = SDK_TRANS_ID,
-                    messageVersion = "2.1.0"
+                    messageVersion = "2.1.0",
```

Comment:
Since this is in the playground only, no need for these to be bumped to 2.2.0?

Reply:
Yeah not necessary but I updated anyways for clarity